src/components/Dropdown/Dropdown.js

```diff
@@ -92,6 +100,12 @@ export default class Dropdown extends PureComponent {
     }
   };
 
+  handleKeydown = evt => {
+    if (evt.which === 27 && this.state.open) {
```

Comment:
Should we use `keyCode` instead of `which` here since `which` is deprecated? Reference: https://developer.mozilla.org/en-US/docs/Web/API/KeyboardEvent/which

Reply:
Hi @joshblack though `which` is deprecated, IIUC it's the only option that works well with all support browsers (in other words, other options require some browser-specific code).

Reply:
Oh duh, good point @asudoh. `keyCode` is also deprecated too lol.